lib/double_entry/reporting.rb

```diff
@@ -78,8 +78,8 @@ class AggregateFunctionNotSupported < RuntimeError; end
     # @raise [Reporting::AggregateFunctionNotSupported] The provided function
     #   is not supported.
     #
-    def aggregate(function, account, code, range, options = {})
-      Aggregate.formatted_amount(function, account, code, range, options)
+    def aggregate(function, account, code, range, filter: nil, partner_account: nil)
```

Comment:
We have 833 downloads for the current version of the gem. It's safe to assume it's in use by many people so we should be considerate. However, we should also be able to make braking changes, coinciding with an increment of the major version number. Let's not go overboard with breaking changes, let's just make solid changes were we need to implement this feature.

It'd be good to note the breaking changes as they're made so it's easier to document them later.